lib/pbench/server/api/resources/upload_api.py

```diff
@@ -264,11 +257,12 @@ def _put(self, args: ApiParams, request: Request, context: ApiContext) -> Respon
             )
 
             current_app.logger.info(
-                "PUT uploading {} for user_id {} (username: {}) to {}",
+                "PUT uploading {} for {} to {}: (access {}, archiveonly {})",
                 filename,
-                user_id,
                 username,
                 tar_full_path,
+                access,
+                metadata.get(Metadata.SERVER_ARCHIVE, False),
```

Comment:
I'm not sure what you mean: "this code" (our dataset `PUT`) is the _only_ definition of the meaning of `server.archiveonly` ... it determines whether we READY the `INDEX` operation. Yeah, I could have just logged the `None` default, which means that the reader of the log needs to understand that's the same as `False`. I decided that making it explicit was better.

Reply:
The operational effect of an unspecified `metadata[Metadata.SERVER_ARCHIVE]` doesn't happen until 200 lines later...that's a pretty big separation.

If you wanted to capture the value of `metadata.get(Metadata.SERVER_ARCHIVE, False)` and use it both here and at line 474, then that would be fine.  But, as the code is, it seems like it would be better to report the input here and not try to predict the effect.

Reply:
That's both more and less information; but if you're going to insist on raw data rather than information, I guess on balance I'm going to just remove both `access` and `archiveonly` here: the values are captured in the NGINX URL already, so showing raw values adds nothing.